### PR TITLE
Updates for Julia 0.6

### PR DESCRIPTION
**Pull request by timholy**
This is against teh/constrained, not master.

### CODE REVIEW

**Comment by pkofod** [comment]
If you say the word I'll merge

**Comment by timholy** [comment]
Word.

**Comment by Evizero** [comment]
> Word.

I saw that comment pop up in my news feed just now (without any context), and I have to admit I was somehow really hoping you were casually using the hiphop-slang of saying "I totally agree with what you are saying" in some conversation 😄

**Comment by timholy** [comment]
I'm not nearly that cool. :smile:

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+environment:
+  matrix:
+    - JULIAVERSION: "julialang/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
```
Comment (Evizero) [inline appveyor.yml]: staticfloat recently made a huge array of automated PRs updating these URLs

```
  matrix:
    - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x86/0.5/julia-0.5-latest-win32.exe"
    - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.5/julia-0.5-latest-win64.exe"
    - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x86/0.6/julia-0.6-latest-win32.exe"
    - JULIA_URL: "https://julialang-s3.julialang.org/bin/winnt/x64/0.6/julia-0.6-latest-win64.exe"
#    - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x86/julia-latest-win32.exe"
#    - JULIA_URL: "https://julialangnightlies-s3.julialang.org/bin/winnt/x64/julia-latest-win64.exe"
```

Not suggesting it needs to be done here right now, but we should keep in in mind. 
I know what changed and can make a PR once this is merged

**Comment by pkofod** [comment]
This thread would have been 36% better if @timholy had actually used "Word." as slang for a general acknowledgement, but... we'll have to do with boring deprecation comments and CI stuff instead... Maybe there's a brighter future where Julians only speak "hip-hop"-speak, and we all divide ourselves into fractions that can be identified by the color of our clothes...

**Pull request merged**